pkgs/by-name/im/imake/package.nix

```diff
@@ -55,9 +55,15 @@ stdenv.mkDerivation (finalAttrs: {
     }\"'";
   };
 
+  preInstall = ''
+    mkdir -p $out/lib/X11/config
+    ln -s ${xorg-cf-files}/lib/X11/config/* $out/lib/X11/config
```

Comment:
Shouldn't we cp them to avoid the extra dependency?

Reply:
`xorg-cf-files` is *just* all the text files that we're linking here. I don't think copying instead of linking to "avoid the extra dependency" makes any sense here if you're having size concerns, and it obscures where these configs actually come from when you look at the resulting `imake` files.